pkg/systemdgen/systemdgen.go

```diff
@@ -0,0 +1,40 @@
+package systemdgen
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"path/filepath"
+)
+
+var template string = `[Unit]
+Description=%s Podman Container
+[Service]
+Restart=%s
+ExecStart=/usr/bin/podman start %s
+ExecStop=/usr/bin/podman stop %s
```

Comment:
we usually add "-t 10" to all our containers to give the container some time to stop properly.

Reply:
Maybe make timeout configurable? Podman generate systemd --timeout <x>?

Reply:
> Maybe make timeout configurable? Podman generate systemd --timeout ?

--stop-timeout then, if we have an option

Reply:
I like that.
I think we should also have --restart-policy and default to on-failure.